index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Weather app</title>
+        <!-- importing stylesheet -->
+        <link rel="stylesheet" href="styles.css">
+        <!-- importing javascript file -->
+        <!-- defer means that the java file will be executed after 
```

Comment:
javascript :)

Reply:
woops, I understand the difference, but I keep making the mistake in text :O, changed